Eazybee/EPIC-MAIL
# #164389956 Setup development environment

#### What does this PR do?
-  Setup development environment

#### Description of Task to be completed?
- Create a file structure for API
- Initialize npm
- Install and configure dev dependencies

#### How should this be manually tested?
- Go to SERVER/v1 folder
- You should see the file and folder structure
- On v1 folder, open package.json file to check installed dependecies

#### Any background context you want to provide?
- This is just to setup the development enviroment, nothing major has been done yet

#### What are the relevant pivotal tracker stories?
- #164389956  (https://www.pivotaltracker.com/story/show/164389956)
